src/main/resources/XWiki/SuggestDisplay.xml

```diff
@@ -258,13 +269,47 @@
     img.observe('click', this.removeItem.bindAsEventListener(this));
     li.appendChild(span);
     li.appendChild(hiddenInput);
+    //Add the edit button
```

Comment:
This code should be fully protected inside the if(inEditItemsMode)
That would be less risky